### DISCUSSION TITLE
[QUESTIION]:immediateFlush and bufferedIO cut into two parts

### DISCUSSION THREAD

**Discussion by 7y-9** [apache/logging-log4j2#2184] [Q&A]
When using log4j2, we found that the last log file was always part of a log, which configuration caused this situation. 
Thanks.

**Comment by vy** [comment]
@7y-9, as a start, could you verify that the issue persists with the latest Log4j version, that is, `2.22.1`?

If upgrading to the latest Log4j version doesn't fix the issue, to help us reproduce the problem, could you provide the following, please?
1. A minimal `log4j2.xml`
2. A minimal `Test.java`
3. Description of what you were expecting
4. Description of what has actually happened
